src/styles/partials/components/_step-list-dark.scss

```diff
@@ -23,10 +23,13 @@
   .dg_step-list.static {
     .dg_step-list__toggle-btn {
       cursor: default;
+      &:focus {
```

Comment:
Not sure if we need this in _step-list-dark.scss as well. When testing locally and only using the outline: none; in _step-list.scss, this seems to still work as expected. I'm testing on mac so my experience could differ though.

Reply:
> Not sure if we need this in _step-list-dark.scss as well. When testing locally and only using the outline: none; in _step-list.scss, this seems to still work as expected. I'm testing on mac so my experience could differ though.

@theilman7 this sounds right to me, typically only color changes are needed in the "theme" file, although i could see this potentially possible. I have not tested though.

Reply:
@theilman7 @martypowell confirmed this is was needed. Good catch Travis

Reply:
Not needed